themes/atomic/atomic.theme.bash

```diff
@@ -166,7 +166,48 @@ ___atomic_prompt_clock() {
 	printf "%s|%s|%s|%s" "${color}" "${info}" "${bold_white}" "${box}"
 }
 
-___atomic_prompt_battery() {
+case "$OSTYPE" in
+  cygwin*) ___atomic_prompt_battery() {
+	[ "${THEME_SHOW_BATTERY}" != "true" ] && return
+	batp=$(echo porcent=$(WMIC PATH Win32_Battery Get EstimatedChargeRemaining /Format:List) | grep -o '[0-9]*')
+	bats=$(echo porcent=$(WMIC Path Win32_Battery Get BatteryStatus /Format:List) | grep -o '[0-9]*')
+
+	if [ "$batp" -gt 50 ]; then
+		color=$bold_green
+	elif [ "$batp" -lt 50 ] && [ "$batp" -gt 25 ]; then
+		color=$bold_yellow
+	elif [ "$batp" -lt 25 ]; then
+		color=$IRed
+	fi
+	
+	box="[|]"
+	[ "$bats" -eq 1 ] && info="-"
+	[ "$bats" -eq 2 ] && info="+"
+	info+=$batp
+	[ "$info" == "+100" ] && info="AC"
+	printf "%s|%s|%s|%s" "${color}" "${info}" "${bold_white}" "${box}"
+	} ;;
+  msys*) ___atomic_prompt_battery() {
+	[ "${THEME_SHOW_BATTERY}" != "true" ] && return
+	batp=$(echo porcent=$(WMIC PATH Win32_Battery Get EstimatedChargeRemaining /Format:List) | grep -o '[0-9]*')
+	bats=$(echo porcent=$(WMIC Path Win32_Battery Get BatteryStatus /Format:List) | grep -o '[0-9]*')
+
+	if [ "$batp" -gt 50 ]; then
+		color=$bold_green
+	elif [ "$batp" -lt 50 ] && [ "$batp" -gt 25 ]; then
+		color=$bold_yellow
+	elif [ "$batp" -lt 25 ]; then
+		color=$IRed
+	fi
+	
+	box="[|]"
+	[ "$bats" -eq 1 ] && info="-"
+	[ "$bats" -eq 2 ] && info="+"
+	info+=$batp
+	[ "$info" == "+100" ] && info="AC"
+	printf "%s|%s|%s|%s" "${color}" "${info}" "${bold_white}" "${box}"
+	} ;;
+  darwin*) ___atomic_prompt_battery() {
 	[ ! -e "$BASH_IT"/plugins/enabled/battery.plugin.bash ] ||
```

Comment:
This code is going to break when I commit the first changes for #944. The name of the plugin in the `enabled` directory is going to be different from then on.

Can't you simply check whether the `battery_percentage` function is defined?